CONTRIBUTING.md

```diff
@@ -105,6 +105,10 @@ Some examples of valid locale tags:
 
 The default language used for pages is English (US). Pages written in English are stored in the default `pages` directory (notice the absence of a specific language tag). Although not strictly required, if you'd like to add a new page in a different language, please consider creating the English page too.
 
+## Inclusive language
+
+Where possible, use inclusive language. In an effort to be clear and consistent, however, this is not always possible. For projects and technologies **that have not** updated their terminology (e.g. master/slave), please use the antiquated terminology as it is in order to preserve the original meaning of the text.
```

Comment:
> I am also not sure about the position of this paragraph. I think it may be better to move it under all the technical instructions, before the "Licensing" paragraph, just to keep things ordered.

I actually disagree here. This page currently has information about the content of the pages first, and then information about how to contribute (opening PRs, license, etc.) at the bottom. To me, the original placement made more sense.

Reply:
Let's unresolve this conversation until @mebeim is able to respond to my counter-suggestion.

Reply:
Fine by me, we can keep it there.

Reply:
@qvissak can you move this back to the original location (between the "Default language for newly added pages" and "Submitting a pull request" sections)? Sorry for the inconvenience.